tests/resources/Nimbus-Util.robot

```diff
@@ -36,14 +36,14 @@ Get IP
     [Return]  ${ip}
 
 Deploy Nimbus ESXi Server
-    [Arguments]  ${user}  ${password}  ${version}=${ESX_VERSION}  ${tls_disabled}=True
+    [Arguments]  ${user}  ${password}  ${version}=${ESX_VERSION}  ${tls_disabled}=True  ${return_pod_id}=False  ${additional-args}=
```

Comment:
I don't like this, not very clean. If you don't want to go through and fix all the tests to handle a multi return (I don't think you should for this since we are probably going to use a testbed in the future anyways) then just simplify this again and use an env var. %{NIMBUS_POD}